### ISSUE TITLE
If providing a large-scale service doesn't work, can we just provide an RSS service?

### ISSUE THREAD

**Issue by mycfctech** [zedeus/nitter#1182]
If providing a large-scale service doesn't work, can we just provide an RSS service?

**Comment by zedeus** [comment]
RSS traffic is heavier than you might expect. @animegrafmays do you think this is feasible?

**Comment by zedeus** [comment]
RSS is cached for 15 minutes by default, so shouldn't matter really. The problem is once you switch to just RSS, the scrapers will follow. nitter.net made about 8 million API requests daily for comparison (~100 per second). Back when RSS was still enabled, 50-100k feeds would be cached at any given moment. At the low end, 50k feeds being updated once every 15 minutes would mean 4.8 million API reqs daily, or 55 per second.

**Comment by markcellus** [comment]
> people who's rss reader is configured to check every 10 seconds or so

Ha. That's being pretty generous. I've seen readers (even the more "mature" ones) make up to 20-30 different requests as the exact same millisecond.

**Comment by iitzRichie** [comment]
> It would be sad to reduce Nitter to only be able to view single tweets (without comments) and the main "Tweets" tab on a profile + RSS, but I suppose it's better than nothing.

For what it's worth (if anything), me personally that's *why* I used Nitter in the first place. I could not care about seeing replies (both from the account itself and comments in reply to the tweet). Just show me the tweet.

**Comment by ctoday1** [comment]
I vote for this solution. Much better than using actual twitter. Please implement.

**Comment by InklingGirl** [comment]
The RSS service was invaluable, if `nitter` can be more sustainably continued by limiting it to that use case, I'm all for it.

**Comment by Slaughterford** [comment]
I, too, would like to see the RSS service continue at the very least.  I'll be honest, I found Nitter long ago because I was searching for a way to convert Twitter posts to an RSS feed for my browser's Live Bookmarks and my phone's RSS Aggregator App.  As great as it was that Nitter allowed us to browse Twitter without ads and bloat, I found the RSS component to be beyond invaluable.  Keeping the RSS function, even without the rest of Nitter, would definitely provide a search that doesn't seem to exist anywhere else out there.

**Comment by jcampbell05** [comment]
Somehow Google serves the tweets so makes sense

**Comment by catigerose** [comment]
I only need  several Twitter RSS feeds .  RSS icached for 30 minutes or more is OK for me.   why is this so hard  since the guy bought Twitter !
vote for this solution.

**Comment by thomas-333** [comment]
In terms of making RSS feeds even better I'm using the following two commits on my private instance:

https://github.com/zedeus/nitter/pull/820
https://github.com/zedeus/nitter/pull/1137

The first one in particular makes RSS much, much better by including quoted tweets in the feed. 

To make the feed perfect it would be great to have a link to videos included, which seems to be more practical than the code necessary for embedded videos to work:

https://github.com/zedeus/nitter/issues/836

**Comment by notliamsi** [comment]
Firstly, a huge thank you to @zedeus and all the contributors to this incredible project. It's individuals like you who ensure we still have an open and free corner on the internet, away from the clutches of corporations and capitalism.

Regarding the RSS service, it would be fantastic to have it available separately, perhaps provided in Nitter and as another repository. This way, those who only want RSS can easily access it.

**Comment by rjolina** [comment]
I fully agree, however, for better readability, (rss)tweets should include the cited content, not just links to it.

**Comment by thomas-333** [comment]
Looking at the issues tracker for RSSHub is seems to me that their Twitter RSS is broken?

**Comment by zedeus** [comment]
Hosting large-scale instances is feasible, and operators can enable RSS if they can manage the traffic. Closing.
